content/docs/start/data-pipelines.md

```diff
@@ -90,11 +90,11 @@ The command options used above mean the following:
   but the idea is that the stage can depend on field values from a parameters
   file (`params.yaml` by default):
 
-```yaml
-prepare:
-  split: 0.20
-  seed: 20170428
-```
+  ```yaml
+  prepare:
+      split: 0.20
+      seed: 20170428
+  ```
```

Comment:
Had to force commit this since Pretty doesn't like it. Formatting check will prob. fail. But this looks correctly for now:

![image](https://user-images.githubusercontent.com/1477535/117697490-220edc80-b188-11eb-88fa-aceb30d4c3b0.png)